projects/amass/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -eu
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+#Complie Amass Binary
+go install ./...
+
+# Based on the function from oss-fuzz/projects/golang/build.sh script.
+
+function compile_fuzzer {
+  path=$1
+  function=$2
+  fuzzer=$3
+
+   # Instrument all Go files relevant to this fuzzer
+  go-fuzz-build -libfuzzer -func $function -o $fuzzer.a $path
+
+   # Instrumented, compiled Go ($fuzzer.a) + fuzzing engine = fuzzer binary
+  $CXX $CXXFLAGS $LIB_FUZZING_ENGINE $fuzzer.a -lpthread -o $OUT/$fuzzer
+}
```

Comment:
Where is the fuzzer, compile_fuzzer needs to be called on an actual fuzzer. A fuzzer can be anything that processes untrusted data. It should ideally live in project repo, e.g. https://github.com/google/gonids/blob/master/fuzz.go#L18
and then called here like
https://github.com/google/oss-fuzz/blob/0d226427c7695564549d394fdb5f24d1dc6bf4d5/projects/gonids/build.sh#L31